File: sqlConnect.php
<?php
//User credentials
$servername = "localhost";
$username = "User";
$password = "password";
$dbname = "school";

// Create connection
$conn = new mysqli($servername, $username, $password, $dbname);

// Check connection
if ($conn->connect_error) {
    die("Connection died: " . $conn->connect_error);
}
echo "Connection success"."<br>";

//Run the sql and create a var for the data returned
$sql = "SELECT name, age, gradeLevel FROM students";
$result = $conn->query($sql);

if ($result->num_rows > 0) {
    // output each row
    while($row = $result->fetch_assoc()) {
        echo "Name: " . $row["name"]. " - Age: " . $row["age"]. " - Grade Level: " . $row["gradeLevel"]. "<br><br>";
    }
} else {
    echo "No data found";
}
$conn->close();
?>

File: sqlConnect.py
#!/usr/bin/python

import MySQLdb

#create the connection
db = MySQLdb.connect(host="localhost", user="User", passwd="password", db="school")

#create cursor
cur = db.cursor(MySQLdb.cursors.DictCursor)

#Create table as per requirement
sql = "SELECT * from students"

#execute the cursor
cur.execute(sql)

#print the rows
rows = cur.fetchall()
for row in rows:
	print(row)
	
#disconnect from server
db.close()
